decidim-comments/spec/forms/comment_form_spec.rb

```diff
@@ -15,13 +15,15 @@ module Comments
       end
 
       let(:organization) { create(:organization) }
+      let(:admin_user) { create(:user, :admin, :confirmed, organization:) }
+      let(:user_manager) { create(:user, :user_manager, :confirmed, organization:) }
       let!(:component) { create(:component, organization:) }
       let(:body) { "This is a new comment" }
       let(:alignment) { 1 }
       let(:user_group) { create(:user_group, :verified) }
       let(:user_group_id) { user_group.id }
 
-      let(:commentable) { create(:dummy_resource) }
+      let(:commentable) { create(:dummy_resource, accepts_new_comments?: false) }
```

Comment:
By default, the resource should allow comments, so this should stay as it was.
Then, check that a normal user can make comments.
After that create a context where the commentable does not accepts comments (with this `let` entry in there). Then check that a normal user cannot create a comment but admins can

Reply:
Done